src/tools/crossgen2/Common/TypeSystem/Ecma/EcmaModule.Sorting.cs

```diff
@@ -10,13 +10,16 @@ namespace Internal.TypeSystem.Ecma
 {
     partial class EcmaModule
     {
-        internal int CompareTo(EcmaModule other)
```

Comment:
Can we just make `CompareTo` here public? It seems like we need to only compare `EcmaModules` anyway.

Modules don't need to do the kind of complex structural comparisons that constructed types have to do (and by extension methods and fields that could be members of the constructed types) - the ClassCodes and comparers are unnecessary.

Reply:
Sure.